### ISSUE TITLE
step3 migrate vm to tomcat issue

### ISSUE THREAD

**Issue by anzxcolinlcc** [GoogleCloudPlatform/migrate-for-anthos-gke#2]
Hi, at step 3
https://github.com/GoogleCloudPlatform/migrate-for-anthos-gke/tree/main/guides/tomcat/3-migrate

migctl migration create petclinic-migration --source my-ce-src --vm-id tomcat-petclinic --intent Image --workload-type tomcat
Migration petclinic-migration was created. Run `migctl migration status petclinic-migration` to see its status.

@cloudshell:~/m4a-petclinic/tomcat$ migctl migration status petclinic-migration
NAME                    APP-TYPE        CURRENT-OPERATION       PROGRESS        STEP                    STATUS  AGE
petclinic-migration     tomcat          GenerateMigrationPlan   [1/3]           CreateCopiedDisks       Running 6s

dont seem to work as expected, 
$ migctl migration status petclinic-migration
NAME                    APP-TYPE        CURRENT-OPERATION       PROGRESS        STEP            STATUS          AGE
petclinic-migration     tomcat          GenerateMigrationPlan   [3/3]           Discovery       Completed       54s

with the yaml generated it is like below
any idea what I have missed?

also this command is not right:
migctl migration update petclinic-migration --main-config petclinic-migration.yaml
Error: unknown flag: --main-config

![Screen Shot 2021-12-15 at 10 41 43 am](https://user-images.githubusercontent.com/4087822/146096769-d394ce94-341b-4795-8453-c89ea1298635.png)

**Comment by eitaneib** [comment]
Hi,

Can you run:
`migctl version`

Make sure that you are running version 1.10.0 as --main-config was included in version 1.10.0

Looking at the screenshot, your output is from a previous version.

The best way would be to delete the migration and reinstall migrate for anthos by running:
``` bash
migctl migration delete petclinic-migration
migctl setup upgrade
```

**Comment by eitaneib** [comment]
Closing due to no activity.
